pkg/ui/server.go

```diff
@@ -32,15 +35,27 @@ type Server struct {
 
 // NewServer creates a new HTTP server implementation that will handle
 // rendering the JVS form using a controller.
-func NewServer(ctx context.Context, cfg *config.UIServiceConfig) (*Server, error) {
+func NewServer(ctx context.Context, uiCfg *config.UIServiceConfig) (*Server, error) {
 	// Create the renderer
-	h, err := render.NewRenderer(ctx, assets.ServerFS(), cfg.DevMode)
+	h, err := render.NewRenderer(ctx, assets.ServerFS(), uiCfg.DevMode)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create renderer: %w", err)
 	}
 
+	kmsClient, err := kms.NewKeyManagementClient(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to setup kms client: %w", err)
+	}
+
+	var cfg config.JustificationConfig
+	if err := cfgloader.Load(ctx, &cfg); err != nil {
+		return nil, fmt.Errorf("failed to load config: %w", err)
+	}
+
+	p := justification.NewProcessor(kmsClient, &cfg)
+
```

Comment:
I think it would make more sense to do this higher up the stack (in main.go) and then update the signature for `NewServer` to accept a processor.

Reply:
ack on it